robin_stringer/test/author_test.js

```diff
@@ -0,0 +1,40 @@
+var chai = require('chai');
+var chaihttp = require('chai-http');
+chai.use(chaihttp);
+var expect = chai.expect;
+
+process.env.MONGOLAB_URI = 'mongodb://localhost/author_test';
+require(__dirname + '/../server');
+var mongoose = require('mongoose');
+var author = require(__dirname + '/../models/author');
+
+describe('author routes', function() {
+  after(function(done) {
+    mongoose.connection.db.dropDatabase(function() {
+      done();
+    });
+  });
+
+  it('should add author details', function(done) {
+    var authorData = {name: 'test author'};
+    chai.request('localhost:3000')
+      .post('/api/author')
+      .send(authorData)
+      .end(function(err, res) {
+        expect(err).to.eql(null);
+        expect(res.body.name).to.eql('test author');
+        expect(res.body).to.have.property('_id');
+        done();
+      });
+  });
+
+  it('should get author details', function(done) {
+    chai.request('localhost:3000')
+      .get('/api/author')
+      .end(function(err, res) {
+        expect(err).to.eql(null);
+        expect(Array.isArray(res.body)).to.eql(true);
+        done();
+      });
+  });
```

Comment:
since the author route is not added in server.js, I am not sure that these would even pass. Do they?